ts/features/design-system/core/DSSelection.tsx

```diff
@@ -140,11 +151,50 @@ const renderCheckboxListItem = () => (
       <Divider />
       <CheckboxListItem
         disabled
-        checked={true}
+        selected={true}
         icon="coggle"
         value="Usa configurazione rapida"
         accessibilityLabel={""}
       />
     </DSComponentViewerBox>
   </>
 );
+
+// RADIO ITEMS
+
+const mockRadioItems = (): ReadonlyArray<NewRadioItem<string>> => [
```

Comment:
what about adding a disabled one too? 
May it be a possible scenario?

Reply:
It's not common, but I've just added an example to the Selection screen.